src/ObjectiveC/ObjCClass.class.st

```diff
@@ -103,31 +103,60 @@ ObjCClass >> conformsToProtocol: protocol [
 	^ self ffiCall: #(BOOL class_conformsToProtocol(self, Protocol *protocol))
 ]
 
+{ #category : #'objective-c run-time' }
+ObjCClass >> convertPointerArrayReturnedBy: selector toInstances: aClass [
```

Comment:
I'm particularly interested in feedback re this method, especially from @estebanlm...